src/utility_components/NumericField.jsx

```diff
@@ -47,7 +47,7 @@ export class NumericField extends Component {
         // track local changes here
         e.preventDefault();
         this.setState({
-            fieldValue: forceNumber(e.target.value),
+            fieldValue: e.target.value,
```

Comment:
I'm kinda curious what this fixes, and how. It seems like `forceNumber()` should handle negative numbers... `new Number('-1')` works as expected.

Reply:
It was blowing up when someone was typing a number; so '-' wouldn't parse, but '-1' does. When starting to enter a negative value `forceNumber` would change it to a 0 before they could type in the next digit.

Also, I was using `forceNumber` in a dumb way.  I was calling it twice, once at the point where someone is entering a value in the local state, and again when state is pushed from the NumericField component: https://github.com/ccnmtl/stats-interactives/pull/162/files#diff-4c4b472aedb717f6d148fbdf2e632174R35 . So eventually the value gets coerced to a number, just not while they are entering it.

Reply:
Ohh i see. Yeah, i think I've solved this by only converting to a number at the 'onBlur' event, not as the user is typing.

Anyways, if this is a quick fix i'll merge it.